app/celery/tasks.py

```diff
@@ -458,7 +458,7 @@ def handle_batch_error_and_forward(
     signed_and_verified: list[tuple[Any, Any]],
     notification_type: str,
     exception,
-    receipt: UUID = None,
+    receipt: Optional[UUID],
```

Comment:
```suggestion
    receipt: Optional[UUID] = None,
```

We can fix the type and keep the default value of `None`